stdlib/public/core/UnsafePointer.swift.gyb

```diff
@@ -586,6 +586,22 @@ extension ${Self} {
     fatalError("unavailable function can't be called")
   }
 }
+
+extension Int {
+  @_transparent
+  @warn_unused_result
+  public init<U>(bitPattern: ${Self}<U>) {
```

Comment:
0, I think. That way it round trips with the {{init(bitPattern:)}} initializers on UnsafePointer.